ui/src/notebooks/pipes/Example/view.tsx

```diff
@@ -2,7 +2,7 @@ import React, {FC, useContext} from 'react'
 import {PipeProp} from 'src/notebooks'
 import {NotebookContext} from 'src/notebooks/context/notebook'
 
-import Panel from 'src/notebooks/components/Panel'
+import NotebookPanel from 'src/notebooks/components/panel/NotebookPanel'
```

Comment:
why so much nesting?

Reply:
Just feels more organized to me, once there are more components in here (minimap, etc.) it will be easier to navigate

Reply:
i get that, even for it, but i'm trying to think of a happy balance between that intent and ease of use. imagine each one of these pipes as being a self contained node module, and now each one needs to have a concept of the internal structure of the notebook component. Maybe the folder structure is fine as it allows the notebook structure to grow, just gotta figure out how to decouple the notebook structure from the pipe so that we can nest these types of things without changing all of the pipes. off the cuff i'm feeling out a structure like:

```
// in src/notebooks/index.ts
export {
  register,
  NotebookPanel
}
export default Notebook
```

half thoughts and maybe outside of this PR, but what about that as a dependency contract between notebook and pipes?

Reply:
I don't have much of an opinion on how importing / exporting happens